packages/studio-ui/src/components/Viewport/defaults.tsx

```diff
@@ -47,7 +47,7 @@ export const VIEWPORTS: ViewportMap = {
       width: 720,
     },
     type: "mobile",
-    css: "aspect-[720/748]",
+    css: "w-[720px] h-[748px]",
   },
   galaxyzflip5unfolded: {
```

Comment:
The behavior for super long aspect ratios (e.g. Galaxy Z Flip5 Unfolded) seems kind of funky. Has product confirmed that that's what they want?

Reply:
Out of curiosity, what feels weird about it for you? I still haven't heard back from Aaron, but I'll ping him again about it

Reply:
I think part of it is that since it is a phone screen, as a user I'd expect to be able to see everything since in my mind a phone screen is smaller than a computer screen.

Reply:
I see. I showed Aaron that specific behavior as well and he's fine with it for now